api/views.py

```diff
@@ -98,4 +126,13 @@ def delivery_log_webhook(request, id):
         return Response(status=404, data={"error": f" Message ID: {id} does not exist"})
```

Comment:
If the 404 status reads would be useful to audit log then audit logging them here could be added and the create_api_commit_message_from_request e.g. parametrized with status parameter to make the failed fetching loggable using it.

Reply:
https://github.com/City-of-Helsinki/notification-service-api/pull/69#discussion_r1846093286